README.md

```diff
@@ -59,11 +61,24 @@ template:
 </ul>
 ```
 
+## more_categories plugin support
```

Comment:
The plugin naming convention that we've been using for migrated plugins is a bit more human-focused. Would you mind changing "more_categories" to "More Categories"? Also, please add a blank line after Markdown headings.